Lesson2/src/main/java/pl/slvd/university/Faculties.java

```diff
@@ -0,0 +1,55 @@
+package pl.slvd.university;
+
+import java.util.Arrays;
+
+import java.util.Locale;
+import java.util.Scanner;
+
+public class Faculties {
+
+    public enum Faculty {
+        INSTRUMENT, VOCAL, THEATRE, CHOREOGRAPHY;
+
+    }
+
+    public static class Speciality {
+
+        public enum Specialty {
+
+            CONCERT_PERFORMER, ACADEMIC_SINGER, MUSICAL_DIRECTOR, CHOREOGRAPHER;
+
+        }
+
+        static void choose() {
+            System.out.println("Welcome to the University of the Arts!\nHere you can study at the faculties: \n");
+            System.out.println(Arrays.toString(Faculty.values()).replace("[", "").replace("]", "")); //displays all faculties recorded in enum
+            System.out.println("What Specialty are you interested in? \n");
+            System.out.println(Arrays.toString(Specialty.values()).replace("[", "").replace("]", "")); //displays all specialities recorded in enum
+            Scanner faculty = new Scanner(System.in);
+            String getSpecialty = faculty.next();
+            getSpecialty = getSpecialty.toUpperCase(Locale.ROOT);
+            switch (getSpecialty) { //choose your speciality
+                case "CONCERT_PERFORMER" -> {
```

Comment:
you can use enum values here, not strings. This will make error visible in case someone change a variable inside of Specialty enum